### PR TITLE
First draft of getting-started content

### PR DESCRIPTION
**Pull request by AnthonyRollins**
Added all content from the[ Getting Started ](https://docs.google.com/document/d/11SIWzGqDDJN4OADASiXywvcu9C8deXZRvTE-cdn-Lh4/edit#heading=h.oewhc96pb3tu) section, which includes;

- 4 markdown files
- subpage navigation
- updated readme.md

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# getting-started/README.md
+# What is a CRM
+
+A Customer Relationship Management, well-known as CRM, is a way to track relationships with customers. Typically, this is done by storing data, such as contact information and other insights about a customer (recent emails, meetings, phone calls, etc.). CRM’s can be found in many forms, but at Cal-ITP, we use a platform named [Hubspot](https://www.hubspot.com/) for our CRM. Within Hubspot are multiple products that allow us to build out our CRM: *service, sales and marketing*. These products and our context for usage are detailed in this guide.
```
Comment (laneymangan) [inline docs/getting-started/README.md:3]: test

Comment (laneymangan) [inline docs/getting-started/README.md:3]: > A Customer Relationship Management, well-known as CRM,

Edit: A Customer Relationship Management software, well-known as a CRM, 

> we use a platform named [Hubspot]

Edit:...we use a platform called [Hubspot]. OR ...we use [Hubspot] 
("named" sounds odd to me...am I being too picky...)

> These products and our context for usage are detailed in this guide.

Edit: These products, and our context for using them, are detailed in this guide.

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -1 +1,7 @@
...
+
+## Purpose
+
+The CRM’s purpose is to increase the effectiveness of Cal-ITP’s relational communication and outreach efforts by establishing transit provider and organization profiles and contact records that detail past and current communication activity such as emails, phone calls, deals (payment pipeline, transit data pipeline, etc.) and support tickets.
```
Comment (laneymangan) [inline docs/getting-started/README.md:7]: I think we should either include the more detailed Purpose section (file 6) here, or remove this section and have the Purpose page directly after 'What is a CRM'

File: docs/getting-started/email-setup.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Email Setup
+
+Logging emails is a central activity to Hubspot and allows for greater transparency and provides an opportunity for team members to have a deeper understanding of where a relationship is at. When configured Hubspot email logging allows to set up automated and/or selective logging of emails while you are in your prefered email tool
```
Comment (laneymangan) [inline docs/getting-started/email-setup.md:3]: > Logging emails is a central activity to Hubspot and allows for greater transparency and provides an opportunity for team members to have a deeper understanding of where a relationship is at.

Edit: Logging emails is central to Hubspot’s ability to track interactions and allows for greater transparency, providing an opportunity for team members to have a deeper understanding of where a relationship is at.

> When configured Hubspot email logging allows to set up automated and/or selective logging of emails while you are in your prefered email tool

Edit: **Once** configured**,** Hubspot....prefer**r**ed email tool**.**

File: docs/getting-started/email-setup.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Hubspot email logging can happen in two ways, you can either set up your email within Hubspot, or you install the [Hubspot email extension](https://knowledge.hubspot.com/connected-email/how-to-install-hubspot-sales). More details on logging will be found in the Activities section.
```
Comment (laneymangan) [inline docs/getting-started/email-setup.md:5]: > Hubspot email logging can happen in two ways, you can either set up your email within Hubspot, or you install the [Hubspot email extension]. More details on logging will be found in the Activities section.

Edit: Hubspot email logging can happen one of two ways; you can either set up your email within Hubspot, or you can install the [Hubspot email extension]. More details on logging can be found in the Activities section.

**Will "Activities Section" be a link once that section is added?

File: docs/getting-started/email-setup.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+For those who don’t have access to add your email to Hubspot, there’s a backup option to ensure communication is continuously tracked within Hubspot. Cal-ITP created, [crm@calitp.org](mailto:crm@calitp.org), to bring all conversations into Hubspot by simply cc’ing this email in any of your communication. Ensure the crm@ email is CC’d, as we want all back and forth conversations to be tracked in the CRM.
```
Comment (laneymangan) [inline docs/getting-started/email-setup.md:7]: > Cal-ITP created, crm@calitp.org, to bring all...

Edit: remove comma's around crm@calitp.org

File: docs/getting-started/email-setup.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+**Note:**: The email set-up function currently does not work for @dot.ca.gov emails – we are actively working with Caltrans IT staff to resolve this issue.
```
Comment (laneymangan) [inline docs/getting-started/email-setup.md:9]: > **Note:**: 

Edit: remove second :

File: docs/getting-started/key-terms.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Key Terms
+
+## Records
+
+Records are the base of Hubspot, and what we build out from, plainly, our records top line is the transit provider or organization and contains all the activity that has occurred with that organization. Creating records enables us to find information on organization or provider’s relationships. Once created, you can store information such as individual contact information, tickets, documents, meeting notes, last activity and any custom properties we have assigned. Records are continually updated and maintained by all users whether that be updating contact information, merged agencies or activity that would be relevant to other user’s understanding and workflows.
```
Comment (laneymangan) [inline docs/getting-started/key-terms.md:5]: >...and what we build out from, plainly, our records top line is the...

Edit:...and what we build out from. Our records top line is the...

File: docs/getting-started/purpose.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# Purpose
+
+The CRM’s purpose is to increase the effectiveness of Cal-ITP’s relational communication and outreach efforts by establishing transit provider and organization profiles and contact records that detail past and current communication activity such as emails, phone calls, deals (payment pipeline, transit data pipeline, etc.) and support tickets. The CRM allows users to understand a provider's relationship to Cal-ITP efforts; with this knowledge, team members can have more coordinated communication and develop stronger relationships to build better outcomes.
```
Comment (laneymangan) [inline docs/getting-started/purpose.md:3]: > relational communication

Edit: ~~relational~~ communication

File: docs/getting-started/README.md (hunk 2 above)
Comment (AnthonyRollins) [reply docs/getting-started/README.md:7]: added the full section to "What is a CRM"

File: docs/getting-started/email-setup.md (hunk 3 above)
Comment (AnthonyRollins) [reply docs/getting-started/email-setup.md:3]: @laneymangan For this edit:
>Edit: Once configured**,** Hubspot....preferred email tool**.**

Did you want me to add 4 Astrix or make that section bold?

File: docs/getting-started/email-setup.md (hunk 4 above)
Comment (AnthonyRollins) [reply docs/getting-started/email-setup.md:5]: Yeah, the activities link will be added once it is added online.

File: docs/getting-started/key-terms.md (hunk 7 above)
Comment (AnthonyRollins) [reply docs/getting-started/key-terms.md:5]: Yeah, @laneymangan, I see what you mean. They are all considered records (contacts, tickets, deals etc.) so you can have your own properties with each. I will need to reword this section all together.

File: docs/getting-started/email-setup.md (hunk 3 above)
Comment (laneymangan) [reply docs/getting-started/email-setup.md:3]: @AnthonyRollins Sorry this one looks crazy lol. Edits are supposed to be: add a comma after configured, add a period after tool and add an r to "prefered". And (definitely optional) change "when" to "once"

Comment (AnthonyRollins) [reply docs/getting-started/email-setup.md:3]: Got it. I've updated that content.

**Comment by AnthonyRollins** [comment]
@thekaveman So, moving the content review to just Laney, I was hoping you could look at my [.pages.yml file](https://github.com/cal-itp/crm-helpdesk/blob/2036703a43eb658fbbe7dcfac47465b9ed5e9f74/docs/getting-started/.pages.yml) as I can't seem to fix the order of my page show up in the 2nd navigation. See [preview page](https://crm-helpdesk-147--cal-itp-previews.netlify.app/getting-started/) for more info, but from my understanding, it should put Purpose on the top, right?

**Comment by thekaveman** [comment]
@AnthonyRollins yes! I have been meaning to get to this... I do see a few issues which should be pretty quick to clean up.

**Comment by thekaveman** [comment]
@AnthonyRollins made the few small changes, looks like it is rendering the way you want now?

**Comment by AnthonyRollins** [comment]
@thekaveman Thanks! This looks good to me. I tried to look at the previous way you wrote the `.pages.yml` (see [example](https://github.com/cal-itp/crm-helpdesk/blob/main/docs/.pages)), and you didn't have `.md` after each page, so I didn't think I need to use it. Unless there is something else you did, but I can't seem to find any other changes you made.

**Comment by thekaveman** [comment]
> and you didn't have .md after each page

Those are actually referencing _folders_ not pages, that's why no `.md` and why it works -- it takes the default page of that folder and directs to that.

The other change is the file is just called `.pages`, no `.yml` extension - although it is a YAML file/format.

**Comment by AnthonyRollins** [comment]
@thekaveman I want to merge this, but I noticed that other PRs have merged before this. Should I re-base this branch before merging, or how would I know it is okay to merge without needing to do that?

**Pull request merged**